include/oneapi/dpl/pstl/utils_ranges.h

```diff
@@ -311,6 +317,8 @@ struct drop_view_simple
 template <typename _R, typename _F>
 struct transform_view_simple
 {
+    using range_value_t = ::std::remove_cv_t<::std::remove_reference_t<std::invoke_result_t<_F, typename _R::range_value_t>>>;
```

Comment:
Why is `::std::remove_cv_t<std::remove_reference_t<...` better than std::decay_t?